### ISSUE TITLE
[🐞] Turning off trailing slash crashes the homepage

### ISSUE THREAD

**Issue by RaeesBhatti** [BuilderIO/qwik#2507]
### Which component is affected?

Qwik City (routing)

### Describe the bug

When I pass `trailingSlash: false` option to `quickCity` plugin in `vite.config.ts`. It crashes the homepage (blank). Restarting the server doesn't fix the problem. If I remove the `trailingSlash` option or turn it to `true`, then it starts to work again. Here is my `vite.config.ts`

```ts
import { defineConfig } from 'vite';
import { qwikVite } from '@builder.io/qwik/optimizer';
import { qwikCity } from '@builder.io/qwik-city/vite';
import tsconfigPaths from 'vite-tsconfig-paths';

export default defineConfig(() => {
  return {
    plugins: [qwikCity({ trailingSlash: false }), qwikVite(), tsconfigPaths()],
    preview: {
      headers: {
        'Cache-Control': 'public, max-age=600',
      },
    },
  };
});

```

### Reproduction

https://stackblitz.com/edit/qwik-starter-jp7gfn?file=vite.config.ts

### Steps to reproduce

Use `{ trailingSlash: false }` as params to `quickCity` plugin in `vite.config.ts`. Like so:
```ts
import { defineConfig } from 'vite';
import { qwikVite } from '@builder.io/qwik/optimizer';
import { qwikCity } from '@builder.io/qwik-city/vite';
import tsconfigPaths from 'vite-tsconfig-paths';

export default defineConfig(() => {
  return {
    plugins: [qwikCity({ trailingSlash: false }), qwikVite(), tsconfigPaths()],
    preview: {
      headers: {
        'Cache-Control': 'public, max-age=600',
      },
    },
  };
});

```

### System Info

```shell
Need to install the following packages:
  envinfo
Ok to proceed? (y) 

  System:
    OS: macOS 13.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 1.88 GB / 64.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    npm: 8.11.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
  Browsers:
    Chrome: 108.0.5359.124
    Firefox: 107.0.1
    Safari: 16.2
    Safari Technology Preview: 16.4
  npmPackages:
    @builder.io/qwik: ^0.15.2 => 0.15.2 
    @builder.io/qwik-city: ^0.0.128 => 0.0.128 
    vite: 3.2.4 => 3.2.4
```

**Comment by zanettin** [comment]
Hi @RaeesBhatti 
Thanks a lot for mentioning this issue 🙏 It seems to be a duplicate of https://github.com/BuilderIO/qwik/issues/2342 and should be fixed with the official launche of qwik-city v0.1.0 🎉 
You can try it also by installing the latest beta if you like.

**Comment by RaeesBhatti** [comment]
Closing in favor of https://github.com/BuilderIO/qwik/issues/2342
